src/librustc/Cargo.toml

```diff
@@ -13,12 +13,12 @@ arena = { path = "../libarena" }
 fmt_macros = { path = "../libfmt_macros" }
 graphviz = { path = "../libgraphviz" }
 log = "0.3"
+owning_ref = "0.3.3"
```

Comment:
So @eddyb pointed this out to me -- I'm not sure if we should add a dependency on this. First off i'm not sure if there are technical implications, but also, it's not under the same license as rust (just MIT, not MIT/Apache2).

cc @brson @alexcrichton